apps/meteor/app/file-upload/server/config/GridFS.ts

```diff
@@ -159,9 +160,11 @@ new FileUploadClass({
 	name: 'GridFS:Uploads',
 
 	async get(file, req, res) {
+		const { query } = URL.parse(req.url || '', true);
 		file = FileUpload.addExtensionTo(file);
+		const contentDisposition = query.contentDisposition === 'inline' ? 'inline' : 'attachment';
 
-		res.setHeader('Content-Disposition', `attachment; filename*=UTF-8''${encodeURIComponent(file.name || '')}`);
+		res.setHeader('Content-Disposition', `${contentDisposition}; filename*=UTF-8''${encodeURIComponent(file.name || '')}`);
```

Comment:
I think we can use the same idea as before here, and create a shared function like `getContentDisposition()`. Just to have the shared logic in a single place.